test/performance/array/distCreate.chpl

```diff
@@ -0,0 +1,70 @@
+use Time;
+use Memory;
+
+use BlockDist;
+use CyclicDist;
+use BlockCycDist;
+
+
+type elemType = int;
+const totalMem = + reduce Locales.physicalMemory();
+
+config param correctness = true;
+config const nElemsSmall = if correctness then 100 else 1000000;
+config const memRatio = .25;
+config const nElemsLarge = ((totalMem*memRatio)/numBytes(elemType)):int;
+config const small = true;
+
+const nElems = if small then nElemsSmall else nElemsLarge;
```

Comment:
Not sure it matters, but I'd take a look at `test/scan/scanPerf.chpl`. I try to copy problem size computations from existing tests, but I don't mind keeping this as-is.